Could not find artifact com.github.springtestdbunit:spring-test-dbunit:jar:1.3.0-SNAPSHOT

Meet below error message when `mvn clean install -U`

[ERROR] Failed to execute goal on project spring-test-dbunit-sample: Could not resolve dependencies for project com.github.springtestdbunit.sample:spring-test-dbunit-sample:jar:1.3.0-SNAPSHOT: Could not find artifact com.github.springtestdbunit:spring-test-dbunit:jar:1.3.0-SNAPSHOT -> [Help 1]

--- a/pom.xml
+++ b/pom.xml
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.github.springtestdbunit.parent</groupId>
 	<artifactId>spring-test-dbunit-parent</artifactId>
-	<version>1.3.0-SNAPSHOT</version>
+	<version>1.3.1-SNAPSHOT</version>
 	<packaging>pom</packaging>
 	<name>Spring Test DBUnit Parent</name>
 	<modules>

--- a/spring-test-dbunit-sample/pom.xml
+++ b/spring-test-dbunit-sample/pom.xml
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.github.springtestdbunit.sample</groupId>
 	<artifactId>spring-test-dbunit-sample</artifactId>
-	<version>1.3.0-SNAPSHOT</version>
+	<version>1.3.1-SNAPSHOT</version>
 	<name>Spring Test DBUnit Sample</name>
 	<properties>
 		<spring.version>4.2.5.RELEASE</spring.version>
@@ -75,7 +75,7 @@
 		<dependency>
 			<groupId>com.github.springtestdbunit</groupId>
 			<artifactId>spring-test-dbunit</artifactId>
-			<version>1.3.0-SNAPSHOT</version>
+			<version>1.3.1-SNAPSHOT</version>
 			<scope>test</scope>
 		</dependency>
 		<dependency>

